lib/posix-user/exportsyms.uk

```diff
@@ -64,3 +64,6 @@ getgrnam_r
 setgrent
 endgrent
 getgrent
+setresgid
```

Comment:
Just a nitpick. This is redefined ([line 59](https://github.com/unikraft/unikraft/blob/85f91161e23af66dd267e9c24b8535bdb7b40127/lib/posix-user/exportsyms.uk#L59)).